docs/products/metrics/concepts/storage-resource-scaling.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Storage and resource scaling
+---
+
+Aiven for Metrics® leverages various storage and resource types to optimize your monitoring experience for both cost and performance. Explores how Aiven uses object storage, disk storage, memory, and compute resources.
+
+## Storage solutions
+
+### Object storage
+
+The primary storage for metrics and metadata, offering scalability and
+cost-effectiveness for long-term retention. Aiven manages this infrastructure,
+ensuring data security and reliability. Metrics are uploaded periodically
+(typically every 2 hours) for historical analysis.
```

Comment:
Metrics are being uploaded to object storage by Receivers but are still available for queries, probably we should mention that detail here (not sure).

Reply:
It this it worth mentioning this. Updated the text.